lib/assertions.js

```diff
@@ -932,20 +932,11 @@ module.exports = expect => {
   });
 
   expect.addAssertion(
-    '<number> [not] to be within <number> <number>',
-    (expect, subject, start, finish) => {
-      expect.argsOutput = output => {
-        output
-          .appendInspected(start)
-          .text('..')
-          .appendInspected(finish);
-      };
-      expect(subject >= start && subject <= finish, '[not] to be truthy');
-    }
-  );
-
-  expect.addAssertion(
-    '<string> [not] to be within <string> <string>',
```

Comment:
Quite liking the de-dupe here :)

Reply:
Yeah, it almost pays for all the additions to `lib` :smirk_cat: